src/components/AdvancedSearch.tsx

```diff
@@ -75,7 +75,7 @@ export const AdvancedSearch = (props: AdvancedSearchProps) => {
   return (
     <>
       {
-        <Accordion alwaysOpen={true} style={{ paddingBottom: "8.5em" }}>
```

Comment:
just remembered that this padding was for making sure the bottom of the Accordion could be seen on the Search Page with the frozen Let's Go button. There was probably a better way to do it though